src/main/resources/application.yaml

```diff
@@ -70,7 +70,7 @@ prosessering:
       milliseconds: 5000
   delete:
     after:
-      weeks: 4
+      weeks: 1
```

Comment:
Er det lang nok tid? F.eks. om det skjer noe nå i mai, hvor det er mange fridager og inneklemte dager?

Reply:
Propertyen gjelder kun for sletting av tasker som er ferdige. Så alle som ikke har status FERDIG blir liggende så lenge de må